packages/aws-cdk-lib/aws-ec2/lib/port.ts

```diff
@@ -188,6 +188,41 @@ export interface PortProps {
  * Interface for classes that provide the connection-specification parts of a security group rule
  */
 export class Port {
+  /** Well-known SSH port (TCP 22) */
```

Comment:
only a nit: do we need "Well-known" in all these?  Cleaner to just have `SSH port (TCP 22)` ?

Reply:
I would have added these at the top of the class/enum JSDoc if they weren't declared alongside the rest of the `Port` class, but I don't think separating them is worth avoiding the repetition